### PR TITLE
FOPTS-1122 Update AWS Usage Reduction Policies (Rightsize)

### PR DESCRIPTION
**Pull request by night-codes**
### Description

Update and standardize AWS Usage Reduction Policies:

AWS Rightsize EBS Volumes
AWS Rightsize RDS Instances
AWS Rightsize Compute Instances

### Issues Resolved

[FOPTS-1122 - AWS Usage Reduction Policies (Jira)](https://flexera.atlassian.net/browse/FOPTS-1122)

### Link to Example Applied Policy

[AWS Rightsize RDS Instances](https://app.flexera.com/orgs/1105/automation/applied-policies/projects/60073?policyId=6495bdfb4466b10001e22bc0)
[AWS Rightsize Compute Instances](https://app.flexera.com/orgs/1105/automation/applied-policies/projects/60073?policyId=6495d42d49169e0001431fa2)
[AWS Rightsize EBS Volumes](https://app.flexera.com/orgs/1105/automation/applied-policies/projects/60073?policyId=6495be824466b10001e22bc2)

### Contribution Check List

- [x] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [x] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

File: cost/aws/rightsize_ebs_volumes/aws_volumes_rightsizing.pt [hunk 1]
```diff
@@ -503,10 +600,12 @@ script "js_combine_cloudwatch_data", type: "javascript" do
       "volumeType": vol_read_data.volumeType,
       "iops": vol_read_data.iops,
       "throughput": vol_read_data.throughput,
-      "iopsAverage": iops_average,
+      "iopsAverage": Math.round(iops_average*100)/100,
```
Comment (Rangelmv23) [inline cost/aws/rightsize_ebs_volumes/aws_volumes_rightsizing.pt:603]: How does this work? Shouldn't it be: Math.round(ios*100)/100, ?

It looks like "iopsAverage" value is rounding itself without existing before

Comment (night-codes) [reply cost/aws/rightsize_ebs_volumes/aws_volumes_rightsizing.pt:603]: In which line is this value rounded?

Comment (Rangelmv23) [reply cost/aws/rightsize_ebs_volumes/aws_volumes_rightsizing.pt:603]: I mean, you're using Math.round(iops_average*100)/100, but what is the value of iopsAverage?

**Review by XOmniverse — APPROVED** [review]
LGTM

**Pull request merged**